### PR TITLE
Refactoring and cleanup to data

### PR DESCRIPTION
**Pull request by ritzalam**
- change keys to use snake_case as right now it is a mix of snake_case and camelCase
- change recent_talking_time to date instead of timestamp
- same with polling

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    bbbevents (1.1.0)
```
Comment (jfederico) [inline Gemfile.lock:4]: As this is not a big change, I would tag it as 1.2.0 or even 1.1.1. I don't think there is need for a major release

**Comment by jfederico** [comment]
BTW, the changes broke one of the tests. Travis is not passing

File: Gemfile.lock (hunk 1 above)
Comment (ritzalam) [reply Gemfile.lock:4]: Changed version to 1.2.0

Comment (kepstin) [reply Gemfile.lock:4]: This is likely break compatibility with our statistics html page generation stuff. If it's breaking compatibility, it should be a major version change.

**Review by jfederico — APPROVED** [review]

**Pull request merged**